lib/web_ui/lib/src/engine/window.dart

```diff
@@ -61,6 +64,7 @@ base class EngineFlutterView implements ui.FlutterView {
     // hot restart.
     embeddingStrategy.attachViewRoot(dom.rootElement);
     pointerBinding = PointerBinding(this);
+    onResize.listen(_didResize);
```

Comment:
Do we need to stop listening on dispose of the view?

Reply:
I'm not sure, since we are closing the stream on dispose anyway. I'll remove the listener too just to be safe.